Potentiometer/CS/README.md

```diff
@@ -0,0 +1,305 @@
+---
+page_type: sample
+urlFragment: potentiometer-sensor
+languages:
+  - csharp
+products:
+  - windows
+  - windows-iot
+  - windows-10-iot-Enterprise
+description: Connect a rotary potentiometer and LED to your Windows 10 IoT Enterprise device.
+---
+
+# Potentiometer sensor
+
+This sample shows how to connect a rotary potentiometer and LED to a Upboard. We use a SPI-based ADC (Analog to Digital Converter) to read values from the potentiometer 
+and control an LED based on the knob position.
+
+## Parts needed
+- [1 LED](http://www.digikey.com/product-detail/en/C5SMF-RJS-CT0W0BB1/C5SMF-RJS-CT0W0BB1-ND/2341832)
+- [1 330 &#x2126; resistor](http://www.digikey.com/product-detail/en/CFR-25JB-52-330R/330QBK-ND/1636)
+- ADC
+    - Upboard
+        - [1 MCP3008 10-bit ADC](http://www.microchip.com/wwwproducts/Devices.aspx?dDocName=en010530) or [1 MCP3208 12-bit ADC](http://www.digikey.com/product-search/en?KeyWords=mcp3208%20ci%2Fp&WT.z_header=search_go)
+        - [1 Voltage-Level Translator Breakout](https://www.sparkfun.com/products/11771)
+- [1 10k &#x2126; Trimmer Potentiometer](http://www.digikey.com/product-detail/en/3362P-1-103TLF/3362P-103TLF-ND/1232540)
+- Upboard
+- 1 breadboard and a couple of wires
+- HDMI Monitor and HDMI cable
+
+## Parts Review
+
+In this sample, you have the option of using either the MCP3002 or MCP3208 ADC (Analog to Digital Converter). 
+The differences between the chips are the number of input channels and resolution. 12-bit resolution is more accurate than the 10-bit options, and the number of channels determines how many different inputs you can read.
+
+Below are the pinouts of the MCP3002 and MCP3208 ADCs. 
+
+| MCP3002                                                              | MCP3008 or MCP3208                                                              |
+| -------------------------------------------------------------------- | -------------------------------------------------------------------- |
+| ![MCP3002 Pinout](../../Resources/MCP3002.png) | ![MCP3208 Pinout](../../Resources/MCP3208.png) |
+
+
+#### Upboard Pinout
+
+![UpBoard pinout](../../Resources/Upboard_Pinout.png)
+
+#### Wiring & Connections
+
+##### MCP3002
+If you chose to use the **MCP3002**, assemble the circuit as follows. Note that the wiper pin (the middle pin on the 10k potentiometer) should be connected to `CH0` on MCP3002. You can also refer to the [datasheet](http://ww1.microchip.com/downloads/en/DeviceDoc/21294E.pdf) for more information.
+
+Detailed connection:
+
+![Overall Schematics](../../Resources/OverallCon-3002.png)
+
+The MCP3002 should be connected as follows:
+
+- MCP3002: VDD/VREF - 3.3V on UPBOARD
+- MCP3002: CLK - "SPI0 SCLK" on UPBOARD

```

Comment:
"SPI0 SCLK" should be "SPI CLK" to match pinout?

Reply:
Made the changes as mentioned.